internal_transfers/actions/src/pipeline.ts

```diff
@@ -11,7 +13,45 @@ import {
 } from "./accounting";
 import { Queryable } from "@databases/pg";
 import { TransactionSimulator } from "./simulate/interface";
+import { AbstractProvider } from "ethers";
+import { getTxDataFromHash } from "./utils";
 
+/** Consumes a transaction hash, fetches corresponding transaction receipt,
+ * writes the receipt to a database, then fetches and returns all previously
+ * unprocessed, but ready, transaction receipts.
+ *
+ * @param db - Connection to a database
+ * @param provider - Ethereum network provider
+ * @param txHash - Ethereum Transaction Hash
+ * @param numConfirmationBlocks transaction finalization block confirmations
+ * @return an array of "finalized" Transaction Receipts
+ */
+export async function preliminaryPipelineTask(
+  db: Queryable,
+  provider: AbstractProvider,
+  txHash: string,
+  numConfirmationBlocks: number = 70
```

Comment:
Nit: In the PR description, you mention this number to be `65`.

Reply:
The services repo uses 65... we need to use something larger (in order to be sure the data is available) -- I have chosen 70.

I was actually thinking to put this as a comment with some reference to the services repo directly in the code. How would you like that?

Reply:
Sounds great. Please do that. Thanks!